lib/src/value/argument-list.ts

```diff
@@ -19,7 +19,18 @@ export class SassArgumentList extends SassList {
    * part of the package's public API and should not be accessed by user code.
    * It may be renamed or removed without warning in the future.
    */
-  readonly id: number;
+  readonly id: number | undefined;
```

Comment:
Document what it means for this to be `undefined`.

Reply:
Updated doc string that `undefined` means host constructed argument list. On the object level we don't really care about special value 0 as that's handled by protofier, in which the id default to 0 if unset.